include/utility.h

```diff
@@ -6,6 +6,15 @@
 #include <sstream>
 #include <string>
 #include <unordered_map>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <arpa/inet.h>
+
+#define MAXLINE 4096 /*max text line length*/
```

Comment:
a bit confused on what this sets the MAX for. does it set the MAX limit for a training request data object (the broken up packets)?

Reply:
Yea, perhaps naming as `BUFFER_SIZE` is more appropriate. The usage of this const is a little overloaded.